embedded-aerospike/src/main/java/com/playtika/test/aerospike/EmbeddedAerospikeTestOperationsAutoConfiguration.java

```diff
@@ -45,10 +45,19 @@ public PackageInstaller aerospikePackageInstaller(
     }
 
     @Bean
-    @ConditionalOnMissingBean
-    public ExpiredDocumentsCleaner expiredDocumentsCleaner(AerospikeClient client,
-                                                           AerospikeProperties properties) {
-        return new ExpiredDocumentsCleaner(client, properties.getNamespace());
+    @ConditionalOnProperty(value = "embedded.aerospike.time-travel.enabled", matchIfMissing = true)
+    public AerospikeExpiredDocumentsCleaner expiredDocumentsCleaner(AerospikeClient client,
+                                                                    AerospikeProperties properties) {
+        return new AerospikeExpiredDocumentsCleaner(client, properties.getNamespace());
+    }
+
+    @Bean
+    @ConditionalOnProperty(value = "embedded.aerospike.time-travel.enabled", matchIfMissing = false)
```

Comment:
add test for `EmbeddedAerospikeTestOperationsAutoConfiguration.java` that checks:
1. when this property is disabled, cleaner is disabled
2. when this property is missing
3. when this property is enabled
See an example for such kind of tests: https://github.com/Playtika/testcontainers-spring-boot/blob/develop/embedded-aerospike/src/test/java/com/playtika/test/aerospike/DisableAerospikeTest.java